sparse_ho/algo/backward.py

```diff
@@ -22,29 +25,62 @@ def __init__(self, use_stop_crit=True, verbose=False):
         self.use_stop_crit = use_stop_crit
         self.verbose = verbose
 
-    def get_beta_jac_v(
+    def compute_beta_grad(
             self, X, y, log_alpha, model, get_v, mask0=None, dense0=None,
             quantity_to_warm_start=None, max_iter=1000, tol=1e-3,
             full_jac_v=False):
-        mask, dense, list_sign = get_beta_jac_iterdiff(
+        """Compute beta and hypergradient with backward differentiation of
+        proximal coordinate descent.
+
+        Parameters
+        ----------
+        X: array-like, shape (n_samples, n_features)
+            Design matrix.
+        y: ndarray, shape (n_samples,)
+            Observation vector.
+        log_alpha: float or np.array, shape (n_features,)
+            Logarithm of hyperparameter.
+        model:  instance of ``sparse_ho.base.BaseModel``
+            A model that follows the sparse_ho API.
+        get_v: callable
```

Comment:
Is it too heavy to rename this: get_grad_outer ? It is clearer to me as it does not rely on mathematical notation

Reply:
Forget about it. The PR is already heavy enough. We can address in another one if you agree.

Reply:
I changed `get_v` for `get_grad_outer`